### PR TITLE
OCPBUGS#7765: Updates for ipa-downloader

### PR DESCRIPTION
**Pull request by abhatt-rh**
[OCPBUGS#7765]: Removes instances of the `ipa-downloader` since only the `coreos-downloader` is supported from 4.10 and later releases.

Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OCPBUGS-7765

Link to docs preview:
[Inspecting logs](http://file.rdu.redhat.com/avbhatt/ocpbugs-7765-ipa/installing/installing_bare_metal_ipi/ipi-install-troubleshooting.html#ipi-install-troubleshooting-bootstrap-vm-inspecting-logs_ipi-install-troubleshooting)

QE review:
 - [x] QE has approved this change.
 
Additional information:
Related to https://github.com/openshift/openshift-docs/pull/57027

### CODE REVIEW

**Comment by abhatt-rh** [comment]
Hi @dtantsur Request you to kindly review this PR. Thanks!

**Comment by abhatt-rh** [comment]
@xltian Requesting your review as you as mentioned as the QA contact for this issue. Thanks!

**Comment by abhatt-rh** [comment]
> @xltian Requesting your review as you as mentioned as the QA contact for this issue. Thanks!

@pamoedom Seeing as you are now the QA contact for this issue, requesting your review. Thanks!

**Comment by pamoedom** [comment]
ACK @abhatt-rh , thanks for the heads up, I've reassigned this one to @jadhaj for double checking.

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 1]
```diff
@@ -14,10 +14,7 @@ bootstrapOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-qemu.<architecture
 clusterOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-openstack.<architecture>.qcow2.gz?sha256=a1bda656fa0892f7b936fdc6b6a6086bddaed5dafacedcd7a1e811abb78fe3b0
 ----
 
-The `ipa-downloader` and `coreos-downloader` containers download resources from a webserver or the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify the following two containers are up and running and inspect their logs as needed:
-
-* `ipa-downloader`
-* `coreos-downloader`
+The `coreos-downloader` container downloads resources from a webserver or the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify that the `coreos-downloader` container is up and running and inspect its logs as needed.
```
Comment (dtantsur) [inline modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc:17]: We could even be more specific: it downloads CoreOS boot artifacts (ISO and kernel/initramfs for PXE).

**Review by dtantsur — APPROVED** [review]

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc (hunk 1 above)
Comment (abhatt-rh) [reply modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc]: @dtantsur PTAL at revised content [here](https://github.com/openshift/openshift-docs/pull/57028/files#diff-ea2985fb173a52686f6612b16fddecbe56fe1b3fdf2424408fa96356249cef22R17). Thank you!

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 2]
```diff
@@ -14,10 +14,7 @@ bootstrapOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-qemu.<architecture
 clusterOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-openstack.<architecture>.qcow2.gz?sha256=a1bda656fa0892f7b936fdc6b6a6086bddaed5dafacedcd7a1e811abb78fe3b0
 ----
 
-The `ipa-downloader` and `coreos-downloader` containers download resources from a webserver or the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify the following two containers are up and running and inspect their logs as needed:
-
-* `ipa-downloader`
-* `coreos-downloader`
+The `coreos-downloader` container includes CoreOS boot artifacts, such as the ISO and kernel or initramfs for PXE. The container downloads resources from a webserver or from the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify that the `coreos-downloader` container is up and running and inspect its logs as needed.
```
[the remark this answers is not in the document]
Comment (abhatt-rh) [reply modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc]: Ack and thanks. Reverting to the original text.

**Review by kelbrown20 — COMMENTED** [review]
/lgtm

**Pull request merged**